package.json

```diff
@@ -1,27 +1,34 @@
 {
-  "name": "xrm-pipeline",
-  "version": "0.0.0",
-  "private": true,
-  "description": "Barnard pipeline with sensible defaults",
+  "name": "xrm-csvw-workflow",
+  "version": "1.0.0",
+  "description": "Barnard59 pipeline with sensible defaults",
+  "type": "module",
+  "main": "index.js",
   "scripts": {
-    "postinstall": "mkdirp output",
-    "start": "dotenv ./start.sh --"
+    "start": "mkdir -p output && barnard59 run -v --pipeline=http://example.org/pipeline/toFile pipelines/main.ttl",
+    "to-file": "mkdir -p output && barnard59 run -v --pipeline=http://example.org/pipeline/toFile pipelines/main.ttl",
+    "file-to-store": "barnard59 run -v --pipeline=http://example.org/pipeline/fileToStore pipelines/main.ttl --variable=ENDPOINT_URL --variable=ENDPOINT_USER --variable=ENDPOINT_PASSWORD",
+    "file-to-store-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/fileToStore pipelines/main.ttl",
+    "to-store-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/toStore pipelines/main.ttl",
+    "fetch": "barnard59 run -v --pipeline=http://example.org/pipeline/fetch pipelines/fetch.ttl --variable=SSH_HOST --variable=SSH_PORT --variable=SSH_USER --variable=SSH_PRIVATE_KEY",
+    "fetch-dev": "barnard59 run -v --pipeline=http://example.org/pipeline/fetch pipelines/fetch.ttl",
+    "test": "stricter-standard"
   },
+  "author": "Zazuko",
+  "license": "MIT",
+  "bugs": {},
   "dependencies": {
-    "@tpluscode/rdf-ns-builders": "^0.4.0",
-    "barnard59": "^0.1.2",
-    "barnard59-base": "^0.1.1",
-    "barnard59-formats": "^0.1.1",
-    "barnard59-graph-store": "^0.1.0",
-    "dotenv": "^10.0.0",
-    "dotenv-cli": "^4.0.0",
-    "rdf-ext": "^1.3.5",
-    "readable-stream": "^3.6.0"
+    "barnard59": "^1.1.1",
+    "barnard59-csvw": "^1.0.0",
+    "barnard59-formats": "^1.2.0",
+    "barnard59-ftp": "^1.0.1",
+    "barnard59-graph-store": "^1.0.1",
+    "barnard59-rdf": "^1.1.1",
+    "jsonld": "^4.0.1",
+    "rdf-ext": "^2.0.1",
+    "rdf-stream-to-dataset-stream": "^1.0.0"
```

Comment:
- `jsonld` is no longer required (was in there as workaround for the buggy version 5.x)
- `rdf-stream-to-dataset-stream` is not required
- `barnard59-ftp` could be removed if the fetch pipeline gets removed

Reply:
Cleaned up in https://github.com/zazuko/xrm-csvw-workflow/pull/16/commits/8da00287991833386d50ba07913e084a2d95ac14